test/DebugInfo/hop_to_executor.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %target-swiftc_driver %s -c -g -Onone -o - -Xllvm -sil-print-debuginfo -emit-sil -parse-as-library -module-name m |  %FileCheck %s
+
+// This test ensures that the hop_to_executor source location matches the end of the do block
+
+// CHECK: %35 = function_ref @swift_asyncLet_finish : $@convention(thin) @async (Builtin.RawPointer, Builtin.RawPointer) -> (), loc {{.*}}:19:3, scope 8
```

Comment:
You're hardcoding the numbering in the CHECK here(%35, scope 8, %10, ...), can you remove all the unnecessary details to make the test more future-proof, and replace the line number with `[[@LINE+X]]`?

Reply:
Ah yes! I forgot about the @LINE directive, thanks!